web/src/pages/apd/new/ApdNew.js

```diff
@@ -0,0 +1,585 @@
+    let indexPrime = array.indexOf(value),
+      newArray = array;
+    if (indexPrime > -1) {
+      newArray.splice(indexPrime, 1);
+      setValue(name, newArray, { shouldValidate: true });
+      setAction(newArray);
+    } else if (indexPrime === -1) {
+      newArray.push(value);
+      setValue(name, newArray, { shouldValidate: true });
+      setAction(newArray);
+    }
+  }
+
+  const createNew = () => {
+    let values = getValues();
+    setIsLoading(true);
+    create(values);
+  };
+
+  const disabled = !isValid || !isDirty || (isDirty && !isValid);
+
+  return (
+    <Fragment>
+      <div className="site-body ds-l-container">
+        <main id="start-main-content">
+          <h1 className="ds-h2 ds-u-padding-top--3">
+            Create a New Advanced Planning Document (APD)
+          </h1>
+          <div className="ds-u-padding-bottom--1 ds-u-border-bottom--2">
+            Complete all the fields below to create your APD.
+          </div>
+          {enableMmis === true ? (
+            <Controller
+              name="apdType"
+              control={control}
+              render={({ field: { onBlur, onChange, value, ...props } }) => (
+                <ChoiceList
+                  {...props}
+                  label="What type of APD are you creating?"
+                  labelClassName="ds-h3 label-header label-extended"
+                  hint={
+                    <Alert
+                      variation="warn"
+                      className="ds-u-margin-y--3 ds-u-margin-right--7"
+                    >
+                      <p className="ds-c-alert__text">
+                        This selection cannot be changed after creating a new
+                        APD.
+                      </p>
+                    </Alert>
+                  }
+                  type="radio"
+                  choices={[
+                    {
+                      label: 'HITECH IAPD',
+                      labelClassName: 'label-extended',
+                      value: 'hitech',
+                      hint: 'Health Information Techology for Economic and Clinical Health Implementation APD',
+                      checked: value === 'hitech'
+                    },
+                    {
+                      label: 'MMIS IAPD',
+                      labelClassName: 'label-extended',
+                      value: 'mmis',
+                      hint: 'Medicaid Management Information System Implementation APD',
+                      checked: value === 'mmis'
+                    }
+                  ]}
+                  onChange={e => {
+                    setApdType(e.target.value);
+
+                    onChange(e);
+                  }}
+                  onBlur={onBlur}
+                  onComponentBlur={onBlur}
+                  errorMessage={errors?.apdType?.message}
+                  errorPlacement="bottom"
+                />
+              )}
+            />
+          ) : (
+            <Controller
```

Comment:
I think you could probably create this without anything in the render or just an empty span or something, then you wouldn't have to show this if mmis is disabled